README.zh-CN.md

```diff
@@ -0,0 +1,211 @@
+# kubectl-ai
```

Comment:
How was this translation generated ?

I expect the README to evolve rapidly in the short term, and if the translation process is manual, I am afraid the translated ones will be outdated soon.

Reply:
These translation files are completed using the Cline plugin. In the future, any changes to the README.md file can be synchronized and completed through Cline plugin in VSCode.

Reply:
Even in Kubernetes, translation is updated manually by each localization team, with English as "upstream" and the other languages as "forks": https://kubernetes.io/docs/contribute/localization/

Reply:
@janetkuo If it's just about translating the README within a GitHub Repo, what would the process be? Or is my current approach incorrect?

Reply:
@appleboy the issue with current workflow is that any changes to README.md will require translations to be updated and as you mentioned, the current workflow will require someone to use `Cline` plugin (this adds an extra step and tool dependency). I would have preferred automated translation workflow that doesn't add any new steps/friction to contributor's workflow. 

Another thing I wanted to highlight is that we have other high priority issues we could use help with. Translation is not one of the biggest burning issue right now. If you would like to contribute more to the project, let me know, I can loop you to more important tasks.